app/models/node.rb

```diff
@@ -186,12 +186,13 @@ def likers
   def latest
     revisions
       .where(status: 1)
+      .order(timestamp: :desc)
       .first
   end
 
   def revisions
     revision
-      .order('timestamp DESC')
+      .order(timestamp: :desc)
```

Comment:
btw this is one of the changes the style guide takes care of. if we implement it will save us a lot of headache.